packages/data-point/examples/entity-request-transform-object.js

```diff
@@ -5,8 +5,7 @@ dataPoint.addEntities({
     url: 'https://swapi.co/api/people/?search=r2',
     options: {
       qs: {
-        // because the key starts with $
-        // it will be treated as a TransformExpression
+        // because the key starts with $, it will be treated as a reducer
```

Comment:
is this change done by you or prettier? the reason I had it in two lines is so its easy to read on mobile

Reply:
Ok, changed it back.